README.md

```diff
@@ -15,14 +15,20 @@ If you want to actually build a font or CSS locally, you'll need some more tooli
 
 ### Build the project
 
-You'll need [`bun`](https://bun.sh) to install development dependencies and build the project.
+You'll need [`bun`](https://bun.sh) to install development dependencies and
+build the project. Check out and build the project as follows:
+
+```shell
+git clone https://github.com/cubing/icons && cd icons
+make
+```
 
 If you don't have `bun` installed, we provide a [`nix`](https://nixos.org/)
-shell you can activate with `nix develop`.
+shell you can activate with `nix develop`:
 
 ```shell
 git clone https://github.com/cubing/icons && cd icons
-nix develop  # Optional. See above.
+nix develop
```

Comment:
I'd rather not treat nix too specially here. People need to get bun installed somehow. They can DIY it, or they can use the dev shell.

Can we just remove the `nix develop` line from the original example? Then you can copy paste without issue.

Reply:
Also works for me.

Reply:
Alright, lemme know if you like 546af643d121379f8a82c16b374bace871e1df6d !